app/qml/dialogs/MMSyncFailedDialog.qml

```diff
@@ -23,16 +23,28 @@ MMComponents.MMDrawerDialog {
   id: root
 
   property string detailedText
+  property bool hasDetails: {
+    const t = (root.detailedText ?? "").trim()
+    return t.length > 0
+  }
```

Comment:
This should do the same trick, JS will convert the string to boolean, empty strings are falsy, see https://developer.mozilla.org/en-US/docs/Glossary/Falsy

```suggestion
  property bool hasDetails: root.detailedText?.trim() ?? false
```